Re-factored test cases and class names

kangzongxian opened the pull request:
This is to make it sound more in line with our
application.

This makes our code base more readable.

kangzongxian committed:
Re-factored test cases and class names

This is to make it sound more in line with our
application.

This makes our code base more readable.

jehousoh commented on src/test/java/seedu/guest/testutil/GuestBookBuilder.java:
```diff
@@ -0,0 +1,34 @@
+/**
+ * A utility class to help with building Guestbook objects.
+ * Example usage: <br>
+ *     {@code GuestBook ab = new GuestBookBuilder().withPerson("John", "Doe").build();}
```
Should this be withGuest?

kangzongxian replied:
Agreed!

jehousoh commented on src/test/java/seedu/guest/testutil/GuestBookBuilder.java:
```diff
@@ -0,0 +1,34 @@
+    /**
+     * Adds a new {@code Guest} to the {@code GuestBook} that we are building.
+     */
+    public GuestBookBuilder withPerson(Guest guest) {
```
Should this be withGuest?

kangzongxian replied:
Agreed!

jehousoh reviewed (changes requested):
Looks good with some minor changes!

kangzongxian committed:
Fix naming issues

jehousoh reviewed (approved):
lgtm!

Lan-Jingbo commented on src/test/java/seedu/guest/storage/StorageManagerTest.java:
```diff
@@ -54,7 +54,7 @@ public void addressBookReadSave() throws Exception {
          * {@link JsonAddressBookStorage} class.
```
```suggestion
         * {@link JsonGuestBookStorage} class.
```

kangzongxian replied:
Agreed!

Lan-Jingbo commented on src/test/java/seedu/guest/storage/StorageManagerTest.java:
```diff
@@ -54,7 +54,7 @@ public void addressBookReadSave() throws Exception {
          * {@link JsonAddressBookStorage} class.
          * More extensive testing of UserPref saving/reading is done in {@link JsonAddressBookStorageTest} class.
```
```suggestion
         * More extensive testing of UserPref saving/reading is done in {@link JsonGuestBookStorageTest} class.
```

kangzongxian replied:
Agreed!

kangzongxian committed:
Fix naming issues for methods

kangzongxian committed:
Fix merge conflicts with master branch

* master:
  Refractor some remaining "Addressbook" to "Guestbook"
  Update JsonGuestBookStorageTest.java
  Update JsonAdaptedGuestTest.java
  Update JsonGuestBookStorageTest.java

# Conflicts:
#	src/test/java/seedu/guest/storage/JsonGuestBookStorageTest.java

Lan-Jingbo reviewed (approved):
LGTM
